src/site/resources/css/site.css

```diff
@@ -71,3 +71,31 @@ h2:hover .anchor, h3:hover .anchor {
   top: 0;
   right: 0;
 }
+
+@media screen and (max-width: 767px) {
```

Comment:
How was the 767 determined?

Desktop looks fine in full screen, but as I was resizing the window to make it smaller I noticed the left jumped from a small snippet of the left side to taking up the full window at about half screen. I was expecting something like this to kick in when the window got smaller than half, but I have never built screens for mobile before.

Reply:
I can make i lower. 769px is the width of a tablet device. In my opinion, the normal design works fine on tablets. But yes, i think it would be better to choose a smaller value there. something like 600px should work better.

Reply:
Due to the width of some of the table you'll get a horizontal scrollbar on some pages. But that problem occurs for 600px and 767px. Therefor I added the wrapper to the normal style. New version is up now.